src/main/java/org/usfirst/frc/team4099/lib/util/Utils.kt

```diff
@@ -13,6 +13,9 @@ object Utils {
      * @param limit     limited magnitude
      * @return          the limited value
      */
+    fun convertSRXUnitstoPulses(unit: Double): Double{
```

Comment:
Style: should be `convertSRXUnitsToPulses` (capitalize To). Also, put a space before that curly brace.

On a different note, perhaps talk to @notjagan about that conversions interface and move this method there? Discuss using ft or wheel revs rather than just encoder revs.

Reply:
That method is never used, so Jagan told us to delete it.